mautrix/util/ffmpeg.py

```diff
@@ -34,7 +36,90 @@ def __init__(self) -> None:
 
 
 ffmpeg_path = _abswhich("ffmpeg")
-ffmpeg_default_params = ("-hide_banner", "-loglevel", "warning")
+ffmpeg_default_params = ("-hide_banner", "-loglevel", "warning", "-y")
```

Comment:
What's the `-y` for 🤔

Reply:
When the filename was the same, the process would just sit forever waiting for a response to the "overwrite y/n" prompt. -y removes the interactive prompt and defaults to "yes"

Reply:
The code looked like you made the filename not be the same so this shouldn't be needed anymore?